docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -7,24 +7,40 @@
 Storm Reference - Storm Commands
 ================================
 
-Storm commands are built-in or custom commands that can be used natively within the Synapse Storm tool / Storm CLI (see :ref:`syn-tools-storm`).
+Storm commands are built-in or custom commands that can be used natively within the Synapse Storm
+tool / Storm CLI (see :ref:`syn-tools-storm`).
 
 .. NOTE::
   
-  The Storm tool / Storm CLI provides a native Storm interpreter and is the preferred tool for interacting with a Synapse Cortex from the CLI.
+  The Storm tool / Storm CLI provides a native Storm interpreter and is the preferred tool for
+  interacting with a Synapse Cortex from the CLI.
 
 
-The pipe symbol ( ``|`` ) is used to send (pipe) the output from a Storm query to any of the Storm commands, and to send the output from a Storm command back to a Storm query.
+The pipe symbol ( ``|`` ) is used to send (pipe) the output from a Storm query to any of the Storm
```

Comment:
The main thing the `|` does is switch back to query syntax after specifying a Storm command + args. You don't need to use it before a Storm command (though you're allowed to and I know it looks clearer).

Reply:
Finally clarified this / wrote it up properly...